src/wallet/wallet.h

```diff
@@ -686,13 +686,47 @@ class COutput
     std::string ToString() const;
 };
 
-class FromAnyTaddr {
+/**
+ * A class representing a BIP-44 account's spend authority, which can be used
```

Comment:
I know that elsewhere we refer to a BIP 44 coin type, but here we do not mean BIP 44 account; that is only one part of the unified account that this is actually selecting from (modulo the legacy account).
```suggestion
  * A class representing a ZIP 316 account's spend authority, which can be used
```

Reply:
Do you mean a ZIP 32 account? ZIP 316 doesn't define accounts independently of ZIP 32; there's no such thing as a ZIP 316 account that is not also a ZIP 32 account.

Reply:
I think that referring to this as a `ZIP 316` account is perhaps correct, because there's a 1:1 mapping between an account ID and a USK.